scripts/figma-synchronise-tokens.sh

```diff
@@ -92,12 +92,13 @@ for i in "${themes[@]}"
 do
    echo "Generating theme: $i"
    generateTokens "$i"
+   capitalizeTheme="$(tr '[:lower:]' '[:upper:]' <<< ${i:0:1})${i:1}"
 
    # Generate colors tokens
    (printf "/**
  * Provides all tokens of a specific theme edited by design team.
  * This file is automatically generated from /scripts/figma-synchronise-tokens.sh.
  * PLEASE DO NOT EDIT HERE
  */
-export default "; echo "${FINAL_RESULT}") > packages/ui/src/theme/tokens/"${i}".ts
+export const console"${capitalizeTheme}"Theme = "; echo "${FINAL_RESULT}") > packages/themes/src/themes/console/"${i}".ts
```

Comment:
actually this script can handle only console theme ?

Reply:
@philibea yep from now I have no way to detect what product is targeted for each theme as there is only console. Later the script will be updated to be able to do the same for any themes.